commit 9be912b554370bf4628a0af3077360ddd2f544d2 (refs/changes/73/3673/2)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2018-03-17T01:36:39+00:00 (1 year, 7 months ago)
    
    TRITON-75 Allow firewall rules to be applied in order of priority
    TRITON-77 Add support for ESP and AH protocols in firewall rules
    Reviewed by: Jordan Hendricks <jordan.hendricks@joyent.com>
    Approved by: Jordan Hendricks <jordan.hendricks@joyent.com>

diff --git a/config/sapi/services/fwapi/service.json b/config/sapi/services/fwapi/service.json
index cf220ff1..0fcddcdf 100644
--- a/config/sapi/services/fwapi/service.json
+++ b/config/sapi/services/fwapi/service.json
@@ -17,9 +17,9 @@
         ]
     },
     "metadata": {
-        "FWRULE_VERSION": 3,
+        "FWRULE_VERSION": 4,
         "SERVICE_NAME": "fwapi",
         "FWAPI_LOG_LEVEL": "info"
     },
     "manifests": {}
-}
\ No newline at end of file
+}
